src/Modal.scss

```diff
@@ -18,22 +18,34 @@
 
   .modal__header {
```

Comment:
you shouldn't need these indented in modal__content, as they should stand alone (in BEM)